storage.js

```diff
@@ -51,7 +51,7 @@ module.exports = function (generate) {
 
   function reconstructKeys(keyfile) {
     var privateKey = keyfile
-      .replace(/\s*\#[^\n]*/g, '')
+      .replace(/\s*#[^\n]*/g, '')
```

Comment:
:fire: I would honestly not fuck with the regex (unless you wrote tight tests).
This is outside the scope of this PR and could just be tidying which has side-effects

Reply:
Yep, just tidying -- unused backslashes are a footgun IMO but I could make this change in another PR. Good call. I think we had a similar discussion about removing unused backslashes a little while ago too: https://github.com/ssbc/ssb-schema-definitions/pull/3#pullrequestreview-317298952

Reply:
true I didn't see the followup till just now.
Yeah think a different PR would be good. It's _probably_ fine, but this is just in the class of things it's quite easy to make mistakes on.